### PR TITLE
#163520963 Allow a User to Get Articles by Author Followed

### PR DESCRIPTION
**Pull request by tejiri4**
#### What does this PR do?
This PR allow a user to get all articles posted by the authors they follow.
#### Description of Task to be completed?

- Create new route
- Create article feed controller
- Write test

#### How should this be manually tested?
- Clone the repo
- Cd into it
- Run npm install
- Run npm test

#### What are the relevant pivotal tracker stories?
[Get Followed Articles](https://www.pivotaltracker.com/story/show/163520963)

### CODE REVIEW

File: server/seeders/20190115151310-demoUser.js [hunk 1]
```diff
@@ -36,16 +36,16 @@ export default {
...
-        id: 'b9aef994-d8ea-4b42-88be-999d57d68891',
-        fullName: process.env.SUPER_ADMIN_FULLNAME,
-        userName: process.env.SUPER_ADMIN_USERNAME,
-        email: process.env.SUPER_ADMIN_EMAIL,
-        password: process.env.SUPER_ADMIN_PASSWORD,
-        authTypeId: process.env.SUPER_ADMIN_AUTHTYPEID,
-        roleId: process.env.SUPER_ADMIN_ROLE_ID,
-        isVerified: true
-      }
+      // {
```
Comment (rafmme) [inline server/seeders/20190115151310-demoUser.js]: Remove this commented object

Comment (tejiri4) [reply server/seeders/20190115151310-demoUser.js]: Fixed

**Review by rafmme — COMMENTED** [review]
LGTM, yet

**Comment by Makwe-O** [comment]
LGTM yet

**Pull request merged**